lxd/storage/drivers/driver_btrfs_utils.go

```diff
@@ -91,38 +91,39 @@ func (d *btrfs) isSubvolume(path string) bool {
 }
 
 func (d *btrfs) getSubvolumes(path string) ([]string, error) {
-	result := []string{}
+	poolMountPath := GetPoolMountPath(d.name)
```

Comment:
This check should be in a separate commit as isn't related to the actual change described in the commit message.

This makes it easier to backport to LTS series.